### ISSUE TITLE
Add a User Login/Register Cycle.

### ISSUE THREAD

**Issue by paeolo** [DimitriTuaz/radio-didou#15]
Paul for backend.
Dim for frontend.

**Comment by paeolo** [comment]
First I should complete issue about storing deezer/spotify credential, in order to set up the db.

**Comment by DimitriTuaz** [comment]
Things to do on the server side : 
- Return an error if a user with the same email address already exists for signup
- Eventually rename the url for signup? Might be confusing....
- Using the `/users/me` to connect the user if there's a cookie, if have sometimes a 401 error code with `"message":"Error verifying token : jwt expired"`
- Is it normal that 4 cookies are set?

**Comment by paeolo** [comment]
* I added the destruction of the cookie during the logout (endpoint /users/logout).
* I increased the jwt expired time from 10mn to 12hours. I think that if the client receive a jwt exprired error, it should detroy the cookie (ie call /users/logout).
* There is only one cookie set, the others fields are cookie's options. HttpOnly make the cookie unreachable by javascript and SameSite=Lax is a (weak) protection agasint CSRF.
* I havent done the first two items i think we can can close this issue and add a new issues for thoses.
Its dev deployed. gg!.
